### PR TITLE
Art registration + art download integration -  bug fix

### PR DESCRIPTION
**Pull request by pjenys0**
Fix integration bug

### CODE REVIEW

File: supernode/services/artworkregister/task.go [hunk 1]
```diff
@@ -562,42 +563,67 @@ func (task *Task) storeFingerprints(ctx context.Context) error {
...
-	if err != nil {
-		return nil, nil, errors.Errorf("failed to get dupe detection result from dd-service %w", err)
-	}
-
-	var fingerprint []float64
-	err = json.Unmarshal([]byte(ddResult.Fingerprints), &fingerprint)
-	if err != nil {
-		return nil, nil, errors.Errorf("failed to parse fingerprints from dd-service result %w", err)
-	}
+	// img, err := file.Bytes()
```
Comment (pjenys0) [inline supernode/services/artworkregister/task.go:566]: don't forget to un-comment this for final change

File: walletnode/services/artworkregister/task.go [hunk 2]
```diff
@@ -428,72 +429,71 @@ func (task *Task) genRQIdentifiersFiles(ctx context.Context) error {
...
 	}
+	content = append(content, signature...)
 
-	js, err = json.Marshal(&symbolIDFile)
+	// read all content of the data b
+	// compress the contente of the file
+	compressedData, err := zstd.CompressLevel(nil, content, 22)
 	if err != nil {
-		return nil, errors.Errorf("failed to marshal identifiers file with signature %w", err)
+		return "", nil, errors.Errorf("failed to compress: ")
```
Comment (pjenys0) [inline walletnode/services/artworkregister/task.go:489]: include err in returned error

File: walletnode/services/artworkregister/task.go [hunk 3]
```diff
@@ -428,72 +429,71 @@ func (task *Task) genRQIdentifiersFiles(ctx context.Context) error {
 		RqFilesDir: task.Service.config.RqFilesDir,
 	})
 
+	log.WithContext(ctx).Debugf("NAM-DEBUG Image hash %x", sha3.Sum256(content))
```
Comment (danileng20) [inline walletnode/services/artworkregister/task.go:432]: Can remove personal name?

**Review by leonidcode20 — APPROVED** [review]

File: walletnode/services/artworkregister/task.go (hunk 3 above)
Comment (leonidcode20) [reply walletnode/services/artworkregister/task.go]: Removed! Sorry I miss that :(

**Review by danileng20 — APPROVED** [review]

**Review by mykytaccode0 — APPROVED** [review]

**Pull request merged**